cartoframes/data/services/isolines.py

```diff
@@ -118,7 +118,7 @@ def _iso_areas(self,
                    resolution=None,
                    maxpoints=None,
                    quality=None,
-                   exclusive=False,
+                   exclusive=True,
```

Comment:
We need to take into account that exclusive=True by default gives priority to the visualization over the geom operation.